brjs-sdk/sdk/libs/javascript/br-test/src/br/test/TimeUtility.js

```diff
@@ -97,7 +97,14 @@ TimeUtility.executeCapturedFunctions = function(nMsToExecuteTo) {
 		capturedFunction = this.pCapturedTimerFunctionArgs[idx];
 
 		if (nMsToExecuteTo == null || capturedFunction[1] <= nMsToExecuteTo) {
+			var l = this.pCapturedTimerFunctionArgs.length;
 			capturedFunction[0]();
```

Comment:
using "L" for a number variable...
![6a00d83451b8c369e2014e5f45e05d970c-800wi](https://cloud.githubusercontent.com/assets/6085260/8499438/e66dbdd8-218a-11e5-9e18-47d66f71a81c.gif)